lib/PrintAsObjC/PrintAsObjC.cpp

```diff
@@ -2037,7 +2037,8 @@ class ModuleWriter {
   }
 
   void writePrologue(raw_ostream &out) {
-    out << "// Generated by " << version::getSwiftFullVersion() << "\n"
+    out << "// Generated by " << version::getSwiftFullVersion(
+      M.getASTContext().LangOpts.EffectiveLanguageVersion) << "\n"
```

Comment:
Does this mean you want it to be something other than `getSwiftFullVersion`? Like obscure the fact that it came from a newer compiler altogether?